content/ui-patterns/empty-states.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Empty states
+---
+
+import {Flex, Box, Text, BorderBox} from '@primer/components'
+
+Empty states appear when there is no content on a page or feature. The blankslate component is used to represent large empty states across our products. These guidelines demonstrate best practices for using the blankslate component and designing empty states. If you're looking for guidelines on implementation, please refer to [Primer CSS](https://primer.style/css/components/blankslate).
```

Comment:
Can we try framing the "why" we use empty states here? The first two sentences feel duplicative. Perhaps just a slight tweak on wording (please edit as you see fit):
"Empty states are used to fill spaces when no content has been added yet, or is temporarily empty due to the nature of the feature. These guidelines demonstrate..."

Reply:
Yes! Sorry missed this in the proofread. Thank you for correcting it.